gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -0,0 +1,960 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
```

Comment:
I don't think this will be apache licensed. We need to check what license to add. Is there a license header in the SC-1.0 code for this?